apps/editor/src/app/panel/linkPageBreakEditPanel.tsx

```diff
@@ -62,7 +62,7 @@ export function LinkPageBreakEditPanel({value, onClose, onChange}: LinkPageBreak
               onChange={styleOption =>
                 onChange?.({
                   ...value,
-                  styleOption,
+                  styleOption: styleOption ?? undefined,
```

Comment:
rsuite update with a breaking change inside a minor release :) rsuite is not using strict typescript mode, they switched typings from || undefined to || null, which caused issues